### PR TITLE
Added Bombchus in Logic

### PR DESCRIPTION
**Pull request by Squidkingdom**
This PR adds a a setting at the bottom of settings panel. As well as adds a bombchus in logic setting that is off by default. Should be accurate with the edge cases of DC and Bombchu Bowling, as well as (coincidental) updated logic for Hot Rodder Goron. (In Dev/5.0 check can be obtained with bombchus if bombchus are in logic.)

### CODE REVIEW

File: script/chests.js [hunk 1]
```diff
@@ -48,14 +48,26 @@ function lens(loc){
     }
 }
 
+function hasboom(){
```
Comment (dombarrieau) [inline script/chests.js]: hasboom should be hasBoom

File: script/chests.js [hunk 2]
```diff
@@ -48,14 +48,26 @@ function lens(loc){
...
+    if(chuInLogic == true){
+    //    alert('chus in logic')
```
Comment (dombarrieau) [inline script/chests.js]: Any particular reason these comments are here?

File: script/chests.js [hunk 3]
```diff
@@ -598,17 +611,17 @@ var dungeons = [
             'Child Shooting Gallery': { isAvailable: function () {
                 return true; }, },
             'Bombchu Bowling 1': { isAvailable: function () {
-                return items.Bombs; }, },
+                return items.Bombchus; }, },
```
Comment (dombarrieau) [inline script/chests.js]: This is not correct. If Bombchus in logic is off, then you need the bomb bag, not chus

File: script/chests.js [hunk 4]
```diff
@@ -598,17 +611,17 @@ var dungeons = [
...
             'Bombchu Bowling 2': { isAvailable: function () {
-                return items.Bombs; }, },
+                return items.Bombchus; }, },
```
Comment (dombarrieau) [inline script/chests.js]: See above

File: script/chests.js (hunk 2 above)
Comment (Squidkingdom) [reply script/chests.js]: forgot to stage that when removing TODO and commented code. Just a debug alert, IMO easier to use than debug. Will Fix..

File: script/chests.js (hunk 1 above)
Comment (Squidkingdom) [reply script/chests.js]: *Sigh* whoops.. Will fix

File: script/chests.js (hunk 3 above)
Comment (Squidkingdom) [reply script/chests.js]: I have no idea why I did that. LOL Will Fix...

File: script/chests.js (hunk 4 above)
Comment (Squidkingdom) [reply script/chests.js]: Will Fix

**Comment by Squidkingdom** [comment]
Alot of this is just stupid mistakes or oversights. New Commit coming by 6:00pm EST

**Comment by Squidkingdom** [comment]
Ok, that commit you made fixes the only other concern I had. Anything else that I Can change?

**Pull request merged**